noir-projects/noir-protocol-circuits/crates/blob/src/blob.nr

```diff
@@ -265,6 +133,31 @@ pub fn evaluate_blob(
     BlobPublicInputs { z: challenge_z, y, kzg_commitment }
 }
 
+// Evaluates each blob required for a block
+pub fn evaluate_blobs(
+    blob_as_fields: [Field; FIELDS_PER_BLOB * BLOBS_PER_BLOCK],
```

Comment:
Same as prev comment, requesting to pluralise to 'blobs', please. Just because I keep misreading things.